### ISSUE TITLE
Highlight elements with errors in accessibility checker (outline)

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#10138]
## Is your proposal related to a problem?

The accessibility checker currently displays all errors as a list, each with links / buttons to scroll to the specific elements with issues. It’s not necessarily clear what element is failing though.

## Describe the solution you'd like

It would be much simpler to understand the UI if there was an outline around the matching elements.

## Describe alternatives you've considered

Annotations. Outlines feel like a reasonable first step though.

## Additional context

- The outline should be built with a web component rather than styling elements directly, so we reduce the risk of causing issues with the page itself
- It should display intelligently for elements that could have very small widths (empty headings or links)
- Displayed outlines should toggle on and off when the accessibility checker is opened / closed
